### DISCUSSION TITLE
Site isn't loading after initial load (deployed in netlify and pointed to custom domain)

### DISCUSSION THREAD

**Discussion by meghsohor** [gatsbyjs/gatsby#28501] [Help]
## Description

My site is deployed in netlify: [meghsohor.netlify.app](https://meghsohor.netlify.app/) which is working fine.
I have added my own domain to the site and setup the DNS from Netlify domain settings. The custom domain is here: [www.meghsohor.com](https://www.meghsohor.com/).

The problem is the site loads first time without any problem but after the initial load when I am at the Homepage and reload the page the site doesn't load and throws the following error in the console:

![image](https://user-images.githubusercontent.com/13042947/101274631-fb5c0c80-37c9-11eb-8da2-35ea34d6e491.png)

## Environment

Node version: 14.15.1
Gatsby CLI version: 2.13.1
Gatsby version: 2.25.3

**Comment by jamo** [comment]
I can't seem to reproduce it, the errors make me think there is/was an issue with Netlifys routing layer. 
Especially if/when the site works with the other domain.

**Reply by meghsohor** [reply]
Can you please try it again after a few hours or a day without clearing the cache? The site works fine first time after I clear the cache and even works always fine if I open in a private window. The problem only occurs when I am in the homepage and don't clear the cache
